libs/damap/src/lib/widgets/license-wizard/license-wizard-list.ts

```diff
@@ -128,7 +128,7 @@ export const perlArtistic2 = {
   name: 'license.wizard.license.perlArtistic2.name',
   priority: 8,
   available: true,
-  url: 'https://opensource.org/licenses/Artistic-2.0',
+  url: 'https://opensource.org/license/artistic-license-2-0-php/',
```

Comment:
This will redirect to the previous URL. Don't think it is needed

Reply:
The changed URL redirects to the old URL. So `https://opensource.org/license/artistic-2.0` is the correct one :)